frontend/src/components/poll/form/index.js

```diff
@@ -67,6 +67,19 @@ export default class extends Form {
   onCancel = () => {
     let cancel = false
 
+    // Nothing added to the poll so no changes to discard
```

Comment:
Common semantics for forms are calling them either "clean" (no changes made by the user) or "dirty" (changes made by the user).

Can you update this variable to `formIsClean`?